playbooks/cluster/openshift/config.yml

```diff
@@ -2,31 +2,74 @@
 - hosts: all
   remote_user: root
   pre_tasks:
+    - name: include_vars
+      include_vars:
+        file: "{{ item }}"
+      with_items:
+        - "{{ playbook_dir }}/../../../vars/global_vars.yml"
+        - "{{ playbook_dir }}/vars/{{ openshift_ver }}.yml"
+
+    - name: set SELinux to permissive mode
+      command: setenforce 1
```

Comment:
This is setting to enforcing mode.  Did you mean ```setenforce 0```? Or is the text wrong?

Reply:
Correct, I'll fix it.